lib/projectImport.js

```diff
@@ -223,8 +224,21 @@ module.exports = {
     }
 };
 
+//include group names in search
+
+/* make strings comparable*/
+function unify(element, adapter) {
+    element = element.toLowerCase().replace(/( |_|)/g,"");
```

Comment:
Is this replacing not already done during import?

Reply:
here i remove the _ , but ' ' was redundant

Reply:
Im not sure why this is needed to convert _ back to " ". Because all names are converted to _